plugins/Live/Controller.php

```diff
@@ -89,6 +97,10 @@ public function getLastVisitsStart()
     {
         Piwik::checkUserHasViewAccess($this->idSite);
 
+        if (!Live::isVisitorLogEnabled($this->idSite)) {
```

Comment:
same here and in other places ideally there was a `Live::checkVisitorLogEnabled`